evalml/utils/gen_utils.py

```diff
@@ -270,3 +270,27 @@ def is_all_numeric(df):
         if dtype not in numeric_and_boolean_dtypes:
             return False
     return True
+
+
+def _convert_nullable_types_wrapper(pd_data):
+    """
+    Converts a pandas data structure that may have extension or nullable dtypes to dtypes that numpy can understand and handle.
+
+    Arguments:
+        pd_data (pd.Series, pd.DataFrame, pd.ExtensionArray): Pandas data structure
+
+    Returns:
+        New pandas data structure (pd.DataFrame or pd.Series) with original data and dtypes that can be handled by numpy
+    """
+    nullable_to_numpy_mapping = {pd.Int64Dtype: 'int64',
```

Comment:
Would we need other extension dtypes here as well, like `pd.Int32Dtype`?

Reply:
Hm, good question! I think theoretically, yes, though it seems like Woodwork only outputs Int/Float64 right now 🤔

Reply:
Seeing https://github.com/alteryx/evalml/issues/1124 though, looks like it would be better to add these just in case :)

Reply:
Actually, yeah, what I said previously seems to hold. Since we go through Woodwork, it gets converted to Int64:
```
import evalml
from evalml import AutoMLSearch
from evalml.data_checks import EmptyDataChecks
import pandas as pd 
X, y = evalml.demos.load_fraud(n_rows=2500)

y = y.astype(pd.Int32Dtype())

automl = AutoMLSearch(problem_type='binary',
                      objective="auto",
                      max_pipelines=3)
automl.search(X, y, data_checks=EmptyDataChecks())
```

Pretty cool. Thanks for pointing this out tho!

Reply:
We don't currently try to optimize and use Int32 vs Int64. From a Woodwork perspective, Int64Dtype is the only one currently used.